checkmate/templates/admin/allow_url.html.jinja2

```diff
@@ -0,0 +1,63 @@
+{% extends "checkmate:templates/admin/base.html.jinja2" %}
+{% block content %}
+
+  <section>
+    <div class="container">
+      {% if messages %}
+        <article class="message is-danger">
+          <div class="message-body">
+            {% for message in messages %}
+              <p>{{ message.detail }}</p>
+            {% endfor %}
+          </div>
+        </article>
+      {% endif %}
+
+      {% if allow_rule %}
+        <article class="message">
+          <div class="message-body">
+            <p>Allowed as {{ allow_rule.rule }} with hash {{ allow_rule.hash }}</p>
+          </div>
+        </article>
+      {% endif %}
+    </div>
+  </section>
+
+  <section>
+    <div class="container">
+      <fieldset class="box mt-6">
+        <legend class="label has-text-centered">Add to Allow List</legend>
+        <form action="{{ request.route_url("admin.allow_url") }}" method="POST">
+          <input type="hidden" name="csrf_token" value="{{ get_csrf_token() }}">
+
+          <div class="field is-horizontal">
+            <div class="field-label is-normal">
+              <label class="label">URL</label>
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <div class="control is-expanded">
+                  <input class="input" name="url" type="text" placeholder="URL" size="100">
```

Comment:
Made it just text because derived `approved_url` we're supposed to be using from the request sheet is not a proper URL.

Reply:
Do we need any type of validation? I can type anything here and it seem to add it to the list.

Reply:
Created a [draft issue](https://github.com/hypothesis/checkmate/pull/934#issuecomment-2553655831) for now